tests/fixtures/falcon_app/app.py

```diff
@@ -0,0 +1,103 @@
+import falcon
+from pathlib import Path
+
+STATIC_IMAGE_PATH = Path(__file__).parent / 'static'
+
+class FalconApp(object):
+    def on_get(self, req, resp):
+        """Handles GET requests on index (/)"""
+        resp.text = """
+    <html>
+        <head>
+            <title>Hello world from Falcon app</title>
+        </head>
+        <body>
+            <h3>Hello world! This is a home page of Falcon app.</h3>
+            <div>
+            <a href='/second_page/'>LINK</a> to second page.
+            </div>
+        </body>
+    </html>"""
+    
+    def on_get_second(self, req, resp):
+        """Handles GET requests on index (/second_page/)"""
+        resp.text = """
+    <html>
+        <head>
+            <title>Hello world from Falcon app</title>
+        </head>
+        <body>
+            <h3>Hello world! This is a second page of Falcon app.</h3>
+            <a href='/image_page/'>LINK to image page</a>
+        </body>
+    </html>"""
+    
+    def on_get_image(self, req, resp):
+        resp.text = """
+    <html>
+        <head>
+            <title>Hello world from Falcon app</title>
+        </head>
+        <body>
+            <h3>Hello world! This is a image page of Falcon app.</h3>
+            <img src="/images/smile.png" alt="smile">
+            <a href="/">LINK to homepage</a>
+        </body>
+    </html>"""
```

Comment:
```suggestion
    </html>
"""
```

Reply:
I know that David, but I was so lazy to fix that and I have hoped, that nobody will notice that :D. But you have one of your last free evenings and you are bored maybe, so I have to fix that .... :). Thanks for your comment ;).